pkg/cli/auth.go

```diff
@@ -24,6 +24,18 @@ const (
 	// ClientAuthSecretKeyFlag is the Client Auth Secret Key Flag
 	ClientAuthSecretKeyFlag string = "client-auth-secret-key"
 
+	// Okta API key flag
+	//RA Summary: gosec - G101 - Password Management: Hardcoded Password
+	//RA: This line was flagged because of use of the word "secret"
+	//RA: This line is used to identify the name of the flag. OktaApiKeyFlag is the Okta API Key Flag.
+	//RA: This value of this variable does not store an application secret.
+	//RA Developer Status:
+	//RA Validator Status:
+	//RA Validator:
+	//RA Modified Severity:
+	// #nosec G101
+	OktaAPIKeyFlag string = "okta-api-key"
```

Comment:
Is this nosec comment necessary?

Reply:
I could not pass the tests if I didn't have that there, so I'm assuming yes?